spec/system/user/user_edits_profile_spec.rb

```diff
@@ -2,6 +2,10 @@
 
 RSpec.describe "User edits their profile", type: :system do
   let(:user) { create(:user) }
+  let!(:profile_field_group) { create(:profile_field_group, name: "Ice Cream") }
+  let!(:profile_field) do
```

Comment:
When not creating fields through the service objects, you should call `Profile.refresh_attributes!` manually, maybe in the `before` block.